### DISCUSSION TITLE
What is an MSU package, and how can I create an .msu file for an MSI?

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#8204] [Questions]
I have seen that WiX supports adding MSU packages. It says an MSU package refers to a Windows update package."

How can I create an MSU package? I couldn't find the documentation on how to create MSU file types.
Can I create or apply an update for an MSI using an MSU package?

**Comment by wmanning** [comment]
MSU are windows update packages.  You don't create them yourself, you get them from microsoft update.
